3p/integration.js

```diff
@@ -337,7 +335,10 @@ function triggerResizeRequest(width, height) {
 }
 
 function triggerRenderStart() {
-  nonSensitiveDataPostMessage('render-start');
+  if (!window.context.isRenderStartSent) {
```

Comment:
Some background: we defined a list of ad networks that call `render-start` themselves. Doubleclick is the only one on the list right now. For ad networks that are not in the list, we'll call `render-start` for them at our best guess. 

However, zhouyx@ found that some other networks actually directly imported `doubleclick.js` (I believe they have business relation with doubleclick). In this case, render-start are called twice. So, it's not `they` called it multiple times, we contributed once.

On the other hand, at this point, we're not sure if we should add those network to the list. So I think it's safer to just keep a state here.